addons/mrp/models/mrp_workorder.py

```diff
@@ -233,6 +259,12 @@ def _compute_allowed_lots_domain(self):
                 treated |= workorder
         (self - treated).allowed_lots_domain = False
 
+    def action_replan(self):
+        productions = self.mapped('production_id')
+        for production in productions:
+            production._plan_workorders(replan=True)
+        return True
```

Comment:
Private attributes (_name, _description, _inherit, …)
Default method and _default_get
Field declarations
Compute, inverse and search methods in the same order as field declaration
Selection method (methods used to return computed values for selection fields)
Constrains methods (@api.constrains) and onchange methods (@api.onchange)
CRUD methods (ORM overrides)
Action methods
And finally, other business methods.